js/bundle.go

```diff
@@ -63,6 +63,13 @@ func (bi *BundleInstance) getExported(name string) goja.Value {
 // NewBundle creates a new bundle from a source file and a filesystem.
 func NewBundle(
 	piState *lib.TestPreInitState, src *loader.SourceData, filesystems map[string]afero.Fs,
+) (*Bundle, error) {
+	return newBundle(piState, src, filesystems, lib.Options{}, true)
+}
+
+func newBundle(
+	piState *lib.TestPreInitState, src *loader.SourceData, filesystems map[string]afero.Fs,
+	options lib.Options, updateOptions bool, // TODO: try to figure out a way to not need both
```

Comment:
Out of curiosity, does the larger refactor help here?

Reply:
Not really, but IMO this is on the "small problems" list of this codebase :sweat_smile:

Reply:
Arguably I can drop the boolean if I extract 

```golang
    err = bundle.getExports(piState.Logger, rt, updateOptions)
	if err != nil {
		return nil, err
	}
```

in each caller - but I am less happy with this idea then with providing the options and a boolean. And unfortunately we *need* the options for other reasons so not providing them is not possible :( .